Build Failure

/home/solaraquarion/pkgbuild/kvirc-git/src/kvirc-git/src/kvirc/kernel/KviIconManager.cpp:1987:2: warning: #warning IMPLEMENT CLEANUP [-Wcpp]
 #warning IMPLEMENT CLEANUP
  ^~~~~~~
[ 55%] Building CXX object src/kvirc/CMakeFiles/kvirc.dir/kernel/KviInternalCommand.cpp.o
[ 55%] Building CXX object src/kvirc/CMakeFiles/kvirc.dir/kernel/KviIpcSentinel.cpp.o
/home/solaraquarion/pkgbuild/kvirc-git/src/kvirc-git/src/kvirc/kernel/KviIpcSentinel.cpp:299:16: error: redefinition of ‘struct xcb_property_notify_event_t’
 typedef struct xcb_property_notify_event_t
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/xcb/xcb.h:193:0,
                 from /usr/include/qt/QtX11Extras/qx11info_x11.h:46,
                 from /usr/include/qt/QtX11Extras/QX11Info:1,
                 from /home/solaraquarion/pkgbuild/kvirc-git/src/kvirc-git/src/kvirc/kernel/KviIpcSentinel.cpp:47:
/usr/include/xcb/xproto.h:1002:16: note: previous definition of ‘struct xcb_property_notify_event_t’
 typedef struct xcb_property_notify_event_t {
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/solaraquarion/pkgbuild/kvirc-git/src/kvirc-git/src/kvirc/kernel/KviIpcSentinel.cpp: In member function ‘virtual bool KviIpcSentinel::nativeEvent(const QByteArray&, void*, long int*)’:
/home/solaraquarion/pkgbuild/kvirc-git/src/kvirc-git/src/kvirc/kernel/KviIpcSentinel.cpp:325:10: error: request for member ‘atom’ in ‘* ev’, which is of non-class type ‘fake_xcb_property_notify_event_t {aka int}’
   if(ev->atom == (kvi_u32_t)kvi_atom_ipc_remote_command)
          ^~~~
make[2]: *** [src/kvirc/CMakeFiles/kvirc.dir/build.make:4695: src/kvirc/CMakeFiles/kvirc.dir/kernel/KviIpcSentinel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:4566: src/kvirc/CMakeFiles/kvirc.dir/all] Error 2

--- a/src/kvirc/kernel/KviIpcSentinel.cpp
+++ b/src/kvirc/kernel/KviIpcSentinel.cpp
@@ -296,7 +296,7 @@ typedef struct
 	kvi_u32_t full_sequence;
 } fake_xcb_generic_event_t;
 
-typedef struct xcb_property_notify_event_t
+typedef struct
 {
 	kvi_u8_t response_type;
 	kvi_u8_t pad0;

